tests/functional/clients/standalone/test_hints.py

```diff
@@ -224,3 +227,25 @@ def test_gem_hints(self, hints_image):
         packages = pkgs.get("/virtual/gempkg/diamonds-2.0")
         assert packages["type"] == "gem"
         assert packages["version"] == "2.0"
+
+
+class TestHintsGo:
+    def test_go_hints(self, hints_image, analyzed_data):
+        hints = {
+            "packages": [
+                {
+                    "name": "kind",
+                    "version": "v0.10.0",
+                    "type": "go",
+                    "license": "Apache2.0",
+                },
+            ]
+        }
+        result = hints_image(hints, "lean")
```

Comment:
can use any image, as the hints file doesn't really do anything. just creates fake entries. so i didn't actually load the go binary in the image.